src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+
+    private double sideA;
+    private double sideB;
+    private double sideC;
+
+    public IsoscelesTrapezoid(double sideA, double sideB, double sideC) {
+        this.sideA = sideA;
+        this.sideB = sideB;
+        this.sideC = sideC;
+    }
+
+    public IsoscelesTrapezoid(double sideA, double sideB, double sideC, String color) {
+        super(color);
+        this.sideA = sideA;
+        this.sideB = sideB;
+        this.sideC = sideC;
+    }
+
+    @Override
+    public double getArea() {
+        return (sideA + sideB) / 2 * Math.sqrt(Math.pow(sideC, 2)
+                - Math.pow((sideA - sideB), 2) / 4);
+    }
+
+    @Override
+    public void draw() {
+        toString();
+    }
+
+    public String toString() {
+        return "Figure: isoscelesTrapezoid, area: " + getArea() + " sq.units, sideA: "
+                + sideA + " units, sideB: " + sideB + " units, sideC: " + sideC
+                + " units, color: " + getColor();
+    }

```

Comment:
you should simply output the characteristics of a figure in `draw()` method

you don't need to override `toString()` at all


**same should be applied in all other places**